src/wallet/rpcwallet.cpp

```diff
@@ -2334,6 +2334,7 @@ UniValue listunspent(const UniValue& params, bool fHelp)
             "      \"address\"   (string) Zcash address\n"
             "      ,...\n"
             "    ]\n"
+            "4. limit            (numeric, optional, default=0) The maximum number of results to return, 0 means no limit.\n"
```

Comment:
> The limit parameter will also be an optional so its current position at the end of the parameter list is fine and does not with upstream

I see that upstream has another boolean parameter already in this position: `include_unsafe`. Could the two interfere (e.g. does upstream's code allow for any numeric values to get cast to boolean, which could be misinterpreted here if a third party expecting the APIs to be identical just passes it right in)?

Reply:
I think there's a lot of just reusing code between coins and hoping it will work :-/ We want to pull in the named argument stuff anyway, right? How urgent is it to include this parameter for 1.1.0? &mdash; can we leave it to 1.1.1 and pull in named arguments first?

Reply:
Problem is named arguments still hasn't made it in, whereas user wants this.